src/test/config/kind/common-values.yaml

```diff
@@ -0,0 +1,47 @@
+DC_APP_REPLACEME:
+  # claim little resources as there are limitations in GitHub actions
+  resources:
+    container:
+      requests:
+        cpu: 20m
```

Comment:
Is this enough for apps to spin up? I know it won't limit the app resources, but does it fit the nodes?

Reply:
Yes, pods start slower but eventually start.